app/views/clients/viewclient.html

```diff
@@ -14,6 +14,17 @@ <h3 class="bolder">{{'label.heading.delete' | translate}}</h3>
 			<button class="btn btn-primary" ng-click="delete()">{{'label.button.confirm' | translate}}</button>
 		</div>
 	</script>
+	<script type="text/ng-template" id="deletesig.html">
+		<div class="modal-header silver">
+			<h3 class="bolder">{{'label.heading.delete' | translate}}</h3>
```

Comment:
Make this sth like `Delete Client Signature`, It is confusing sometimes when the modal opens up just with `delete` user might not be sure what they are deleting